src/ImGui.NET/ImGuiNative.cs

```diff
@@ -71,10 +71,6 @@ public static unsafe class ImGuiNative
         [DllImport(cimguiLib)]
         public static extern DrawList* igGetWindowDrawList();
         [DllImport(cimguiLib)]
-        public static extern NativeFont* igGetWindowFont();
```

Comment:
You assume correctly. I don't think it's worth it: it's in imgui's release log/documentation (one can hardly work without it) and it's a simple find&replace. The CollapsingHeader obsolete case is worse (btw. if no one uses it yet, I'd rather drop it).